stable/external-dns/templates/deployment.yaml

```diff
@@ -424,10 +425,6 @@ spec:
           mountPath: /transip
           readOnly: true
         {{- end }}
-        # Extra volume mount(s)
```

Comment:
I don't understand what is the problem of the extra volumes could you clarify why are you deleting them?